docs/geozarr-eopf-specification.qmd

```diff
@@ -0,0 +1,304 @@
+
+GeoZarr V3 extends the Zarr V3 format to support geospatial data with proper coordinate reference systems, multiscale overviews, and CF-compliant metadata, specifically addressing the gaps found in EOPF Zarr format.
+
+## Gaps in EOPF Zarr Format
+
+The current EOPF Zarr stores have several gaps that prevent them from being fully GeoZarr compliant:
+
+### Issues Identified
+
+| Issue | Description | Impact |
+|-------|-------------|---------|
+| **Missing CF Standard Names** | Variables lack `standard_name` attributes | Reduces interoperability with CF-compliant tools |
+| **Incomplete CRS Information** | CRS stored only as `proj:epsg` attribute | Limited compatibility with rioxarray and geospatial tools |
+| **Missing Grid Mapping** | No `grid_mapping` attribute linking to spatial reference | Geospatial tools can't detect coordinate system |
+| **No Multiscale Support** | Lacks overview levels and multiscale metadata | Poor performance for multi-scale visualization |
+| **Missing Coordinate Arrays** | Overview levels lack proper x/y coordinate arrays | Cannot perform spatial operations on overview data |
+| **Inconsistent Geotransform** | Geotransform not in CF-compliant format | Spatial referencing may not work across tools |
+
+## GeoZarr V3 for EOPF - Core Requirements
+
+### 1. CF Convention Compliance
+
+All GeoZarr V3 datasets MUST follow CF (Climate and Forecast) conventions:
+
+```json
+{
+  "standard_name": "toa_bidirectional_reflectance",
+  "grid_mapping": "spatial_ref"
+}
+```
+
+**Requirements:**
+- All data variables MUST have a `standard_name` attribute
+- All geospatial data variables MUST reference a spatial reference system via `grid_mapping`
+
+### 2. Enhanced Spatial Reference System
+
+Every GeoZarr V3 dataset MUST include a `spatial_ref` coordinate variable with comprehensive CRS information:
+
+```json
+{
+  "spatial_ref": {
+    "attrs": {
+      "crs_wkt": "PROJCS[\"WGS 84 / UTM zone 32N\"...]",
+      "spatial_ref": "PROJCS[\"WGS 84 / UTM zone 32N\"...]", 
+      "epsg": 32632,
+      "proj4": "+proj=utm +zone=32 +datum=WGS84 +units=m +no_defs",
+      "GeoTransform": "300000.0 60.0 0.0 5000040.0 0.0 -60.0"
+    }
+  }
+}
+```
+
+**Requirements:**
+- MUST include `crs_wkt` in Well-Known Text format
+- SHOULD include `epsg` code when available (preserving EOPF's `proj:epsg`)
+- SHOULD include `proj4` string for additional compatibility
+- MUST include `GeoTransform` in GDAL format for pixel-to-coordinate transformation
+
+### 3. Coordinate Arrays for All Levels
+
+All GeoZarr V3 datasets MUST include proper coordinate arrays at every resolution level:
```

Comment:
if we require `GeoTransform` then we don't really need coordinate arrays. IMO having those array at each level would be painful to handle.